### PR TITLE
feat: visual distinction between instructions and code

### LINKED ISSUE #45352: Suggestion for beta Responsive web course: format instructions
I’m not sure where the right place is to give this feedback. I was going to recommend the beta course to a friend so I decided to try it out.

In general it looks great! Congratulations.

One thing I didn’t like is the lack of visual distinction between code and instructions. For example:

image

I would suggest making the instruction area have a different background colour. Otherwise it’s just a whole lot of gray. I know that the instructions are in a serif font and the code is in a mono font, but to me that isn’t enough to separate. Having the instructions embedded in the code is probably a good thing, but I’d make it more distinct.

Perhaps like this:

image

I'd also recommend that the instructions be indicated with perhaps a heading of ***To Do***. 

Any way, it looks really great and I'm excited to recommend it to friends. Thanks for all the hard work!

### PR DESCRIPTION
**Pull request by kai-wei-mo**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #45352

Following up on https://github.com/freeCodeCamp/freeCodeCamp/issues/45352#issuecomment-1064096301
- Darkened border color of upper and lower jaws
- Added Monaco selected line border color
image

### CODE REVIEW

File: client/src/templates/Challenges/classic/editor.tsx [hunk 1]
```diff
@@ -178,7 +178,8 @@ const defineMonacoThemes = (
     inherit: true,
     // TODO: Use actual color from style-guide
     colors: {
-      'editor.background': options.usesMultifileEditor ? '#eee' : '#fff'
+      'editor.background': options.usesMultifileEditor ? '#eee' : '#fff',
+      'editor.lineHighlightBorder': '#002ead'
```
Comment (ahmadabdolsaheb) [inline client/src/templates/Challenges/classic/editor.tsx:182]:
```suggestion
      'editor.lineHighlightBorder': '#cee8fc'
```

Comment (ahmadabdolsaheb) [reply client/src/templates/Challenges/classic/editor.tsx:182]: Also let's set the border color for the vs-dark-custom theme to :`#0e4470`

Other than that this is LGMT

**Comment by kai-wei-mo** [comment]
Okay I implemented your suggestion. Ready for review and merge :+1:

**Review by raisedadead — APPROVED** [review]

**Comment by ahmadabdolsaheb** [comment]
Thank you for making the requested changes.

**Review by ahmaxed — APPROVED** [review]

**Pull request merged**